ntt/ntt-cooley-tukey.expect

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
If naive and `cooley-tukey` generate the same outputs, then there should be one expect file. Use the `expect_dir` option in `runt.toml` to specify that both tests use the same expect file. You'll have to rename the Dahlia file to have the same name (put the two implementation in new folders).

Reply:
Just to confirm, you're saying have two folders `cooley-tukey/ntt.fuse` and `naive/ntt.fuse` and then place `ntt.data` in the path designated by `expect_dir`?